python/nnvm/frontend/darknet.py

```diff
@@ -0,0 +1,668 @@
+"""
+Compile DarkNet Models
+====================
+**Author**: `Siju Samuel <https://github.com/siju-samuel/>`_
```

Comment:
remove specific author names since it is indicated by the contributor log.  

Darknet frontend